test/test_torch.py

```diff
@@ -13367,6 +13367,35 @@ def test_cat_big(self, device):
         result = torch.cat(concat_list)
         self.assertEqual(result.size(0), SIZE1 + SIZE2)
 
+    @onlyCUDA
+    def test_atomic_add(self, device):
+        # todo: we should probably expose a dtype.is_signed()
+        def is_signed(dtype):
+            return torch.is_signed(torch.tensor(0, dtype=dtype))
+        # https://github.com/pytorch/pytorch/issues/29153
+        for dtype in torch.testing.get_all_math_dtypes(device):
+            size = [5, 5]
+            if dtype.is_floating_point:
+                tensor = torch.rand(size, dtype=dtype, device=device)
```

Comment:
This line does nothing, right? All dtypes are either signed or unsigned and the value here is overwritten in one of the two branches that comes next.